content/en/releases/release.md

```diff
@@ -136,7 +136,7 @@ The general labeling process should be consistent across artifact types.
 
 ## The Release Cycle
 
-![Image of one Kubernetes release cycle](release-cycle.png)
+![Image of one Kubernetes release cycle](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release-cycle.png)
 
 Kubernetes releases currently happen approximately four times per year.
```

Comment:
Also won't we need to change this to `three times per year`? (the image linked also needs to be updated to reflect that there are now three releases in a year).

Reply:
Would you be willing to update https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release.md and then copy that change here using the automation that's in place?

We prefer to make fixes upstream.

Reply:
Sure, will update here once I've made the change on k/community :)